src/standard_library/array_methods.md

```diff
@@ -0,0 +1,139 @@
+
+# Array
+
+For convenience, the STD provides some ready-to-use, common methods for arrays[^migrationNote]:
+
+## len
+
+Returns the length of an array
+
+```rust
+fn len(_array: Self) -> comptime Field
+```
+
+example
+
+```rust
+// arr = [42, 42]
+fn main(arr : pub [Field; 2]) {
+    let t = arr.len();
+    constrain t == 2;
+}
+```
+
+## sort
+
+Sorts the array with a built-in function
+
+```rust
+fn sort(_array: Self) -> Self
+```
+
+example
+
+```rust
+// arr = [42, 32]
+fn main(arr : pub [u32; 2]) {
+    let t = arr.sort();
+    constrain t == [32, 42];
```

Comment:
We should use better variable names in these docs. Instead of `t` for example we can use `sorted` which communicates that sort returns a new array that is in sorted order.